app/assets/stylesheets/business.scss

```diff
@@ -22,6 +22,13 @@ $site-red: #e56c69;
   padding-top: 10px;
 }
 
+.hero-unit8 {
+  background-image: image-url("menu.png");
+  background-size: cover;
+  width: 100%;
```

Comment:
Properties should be ordered background-image, background-size, padding-top, width